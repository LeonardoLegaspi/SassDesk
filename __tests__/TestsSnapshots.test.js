var sass = require('node-sass');

describe('-----VARIABLES TEST', ()=>{
    test('Variables snapshot', async () => {
  
        var result = sass.renderSync({
          data: `
          @import './1_helpers/variables';
          `,
          outputStyle: 'nested'
        });
        
        expect(result.css.toString()).toMatchSnapshot();
      });
})

describe('-----GENERICS TEST', ()=>{
  test('Generics snapshot', async () => {

      var result = sass.renderSync({
        data: `
        @import './2_base/generics';
        `,
        outputStyle: 'nested'
      });
      
      expect(result.css.toString()).toMatchSnapshot();
    });
})