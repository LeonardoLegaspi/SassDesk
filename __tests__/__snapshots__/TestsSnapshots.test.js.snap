// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`-----GENERICS TEST Generics snapshot 1`] = `
"body {
  margin: 0px;
  padding: 0px; }

html {
  font-size: 13px; }
  @media only screen and (min-width: 680px) {
    html {
      font-size: 15px; } }
  @media only screen and (min-width: 1087px) {
    html {
      font-size: 18px; } }

body {
  font-family: Arial, sans-serif; }
"
`;

exports[`-----VARIABLES TEST Variables snapshot 1`] = `""`;
